### PR TITLE
added deploy instruction for application

### PR DESCRIPTION
**Pull request by elainekamlley**
Wrote up some quick deployment instructions. There are more direct commands so let me know if I should add those. As a practice however, I always like to add the long form so folks know exactly what is happening at each step.  Please let me know if you have any feedback. @msecret @bsweger

### CODE REVIEW

File: app/DEPLOY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+1. Set up Cloud Foundry on your computer. Follow the documentation [https://docs.18f.gov/getting-started/setup/](https://docs.18f.gov/getting-started/setup/)
+2. Set up your account by [creating an issue in the DevOps issue tracker](https://github.com/18F/DevOps/issues/new). 
+3. Log in: 
+    ```	
+    cf api https://api.18f.gov
+    cf login
+    ```
+4. Enter you username (gsa email) and password
+5. Select org 2. data-act
```
Comment (msecret) [inline app/DEPLOY.md]: I think I'd like to know the actual command here and for the one below.

File: app/DEPLOY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+6. Select space - 3. pilot-dev
+7. Make sure you are in the app folder of the data-act
```
Comment (msecret) [inline app/DEPLOY.md]: Do we need to add steps to set the env vars for the username, password?

Comment (elainekamlley) [inline app/DEPLOY.md]: Thanks! I updated the .md as per your feedback.

**Comment by msecret** [comment]
Looks great, I'm going to wait to merge until @bsweger has checked in out.

**Comment by bsweger** [comment]
@elainekamlley This looks great. One thing I'd add is a note that the deploy instructions are for people who can access 18F's CF. We've told folks outside of 18F/govt. that they can track the pilot progress here, so we should be clear that this deployment strategy is for the internal team.

**Comment by msecret** [comment]
Good call @bsweger !

**Comment by elainekamlley** [comment]
@msecret @bsweger - How does this look?

**Comment by bsweger** [comment]
Thanks, @elainekamlley

**Pull request merged**